rasa/telemetry.py

```diff
@@ -648,6 +648,7 @@ def initialize_error_reporting() -> None:
             asyncio.CancelledError,  # an async operation has been cancelled by the user
             # expected Rasa errors
             RasaException,
+            OSError,  # user has no space left on device
```

Comment:
This exception class is much broader than just no space left on device.

Reply:
I only listed an example to keep the comment short, I can either remove the comment or just ignore `OSError(errno=28)`, although I thought all types of OSError should be caught?

Reply:
Does specifying the `errno` like that here work? We initially thought you could only do the whole class, but I think we probably want to ignore it all anyway tbh. 
I would either remove the comment, or put a more general comment.

Reply:
I think `OSError` makes sense. We have ton of errors caused by thing like `FileNotFoundError` / `IsADirectoryError` in [here](https://sentry.io/organizations/rasahq/issues/?project=2801673&query=is%3Aunresolved&sort=freq&statsPeriod=30d) which create the vast majority of Sentry events.